Source/DafnyDriver/Commands/CommandRegistry.cs

```diff
@@ -30,34 +30,45 @@ static class CommandRegistry {
     PrintOption.Instance,
     ResolvedPrintOption.Instance,
     BoogiePrintOption.Instance,
+    InputsOption.Instance
   });
 
   static void AddCommand(ICommandSpec command) {
-    Commands.Add(command.Name, command);
+    Commands.Add(command);
   }
 
   static CommandRegistry() {
     AddCommand(new VerifyCommand());
     AddCommand(new RunCommand());
     AddCommand(new BuildCommand());
     AddCommand(new TranslateCommand());
+
+    FileArgument = new Argument<FileInfo>("file", "input file");
+    FileArgument.AddValidator(ValidateFileArgument());
+
+    FilesArgument = new Argument<IEnumerable<FileInfo>>("file", "input files");
+    FilesArgument.AddValidator(ValidateFileArgument());
+  }
+
+  public static Argument<FileInfo> FileArgument { get; }
+
+  private static ValidateSymbolResult<ArgumentResult> ValidateFileArgument() {
```

Comment:
Where is this method used?

Reply:
On line 47 and 50

Reply:
Funny enough, I was looking for `ValidateSymbolResult` instead of `ValidateFileArgument`. Might be because of the forming habit of having the first identifier being the function's name, like in Dafny.